src/components v2/Molecules/Components/FiguresCard/FiguresCard.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="photon-figures-card" :class="[cardStyle]">
+    <div class="figures-card-text-container">
+      <div class="figures-card-title">
+        {{ title }}
+        <slot name="title" />
+      </div>
```

Comment:
Question:

should there maybe be a conditional to check if the title has a value and then expose the slot, in incase both have values. Saying that I don't know enough about its use case on OB-web